rhel8/kickstart/ssg-rhel8-pci-dss-oaa-ks.cfg

```diff
@@ -0,0 +1,163 @@
+# SCAP Security Guide PCI-DSS profile kickstart for Red Hat Enterprise Linux 7 Server
```

Comment:
There is `oaa` in the name of the file, I think it is not necessary and it isn't consistent with the OSPP kickstart file name.